lib/pages/home/recent_notes.dart

```diff
@@ -121,6 +129,49 @@ class _RecentPageState extends State<RecentPage> {
       floatingActionButton: NewNoteButton(
         cupertino: cupertino,
       ),
+      persistentFooterButtons: 
+        <Widget>[
+          ValueListenableBuilder(
+            valueListenable: selectedFiles,
+            builder: (context, selectedFiles, child) {
+              return Collapsible(
+                axis: CollapsibleAxis.vertical,
+                collapsed: selectedFiles.length != 1,
+                child: RenameNoteButton(existingPath: selectedFiles.length == 1 ? selectedFiles[0] : '')
+              );
+            },
+          ),
+          ValueListenableBuilder(
+            valueListenable: selectedFiles,
+            builder: (context, selectedFiles, child) {
+              return Collapsible(
+                axis: CollapsibleAxis.vertical,
+                collapsed: selectedFiles.length != 1,
```

Comment:
You've updated the `move` function but hidden the move button if there's multiple files.
Was this intentional?

Reply:
This was to ensure that the parent folder is always the same. Now different parent folders are possible, so I changed it to be like in the "Browse" screen.